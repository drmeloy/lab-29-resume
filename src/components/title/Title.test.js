import React from 'react';
import { shallow } from 'enzyme';
import Title from './Title.jsx';

describe('Title component', () => {
  it('renders a Title component', () => {
    const wrapper = shallow(<Title title={''} />);
    expect(wrapper).toMatchSnapshot();
  })
});
