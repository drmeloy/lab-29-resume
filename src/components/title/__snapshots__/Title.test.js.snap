// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component renders a Title component 1`] = `<h3 />`;
